hub/powertoys/environment-variables.md

```diff
@@ -0,0 +1,49 @@
+---
+title: PowerToys Environment Variables for Windows
+description: Environment Variables is a convenient way to manage environment variables.
+ms.date: 10/24/2023
+ms.topic: article
+no-loc: [PowerToys, Windows, Environment Variables, Win]
+---
+
+# Environment Variables
+
+Environment Variables app offers a nice and easy way to manage environment variables. Also, you can create profiles and manage multiple variables at once. Profile variables have precedence over User and System variables. In the background, applying the profile adds variables to User environment variables. When profile is applied, if there is an existing User variable with the same name, backup variable is created in User variables which will be reverted to original one on profile un-apply.
+
+Applied variables list shows the current state of the environment, respecting the order of evaluation of environment variables (Profile -> User -> System). Evaluated Path variable value is shown at the top of the list. 
+
+![PowerToys Environment Variables screenshot.](../images/powertoys-environment-variables.png)
+
+## Edit/Remove variable
+
+To edit or remove a variable (profile, User or System) for specific variable select more options button (**•••**):
```

Comment:
```suggestion
To edit or remove a variable (profile, User or System) for a specific variable, select the more options button (**•••**):
```

Using "variable" twice in the sentence doesn't feel correct to me. Could the sentence be changed to start with "To edit or remove a value"?

Reply:
Changed to `To edit or remove a variable (profile, User or System), select the more options button (**•••**) on the desired variable:` to align with profile on line 35

Reply:
does that sound better?